### PR TITLE
Add convenient commands for vim and sarif

### LINKED ISSUE #26: SARIF and vim formats copyable commands
Add the copyable commands for the semgrep sarif and vim formats. As a reader, you'll need to run semgrep scan --help and then read all that to find what you want. Leaving the commands directly to be copied would make it much easier.

### PR DESCRIPTION
**Pull request by ahpaleus**
Closes #26

### CODE REVIEW

File: content/docs/static-analysis/semgrep/00-installation.md [hunk 1]
```diff
@@ -211,9 +211,18 @@ according to your preferences and tooling.
      - **SARIF format**: Use the SARIF format with the Visual Studio Code and the
        [SARIF Explorer](https://marketplace.visualstudio.com/items?itemName=trailofbits.sarif-explorer) extension.
        This makes it easy to review the analysis results and drill down into specific issues to understand their
-       impact and severity.
+       impact and severity. Example usage of the `p/default` ruleset with the SARIF output file:
```
Comment (fcasal) [inline content/docs/static-analysis/semgrep/00-installation.md:214]: output file -> output format

File: content/docs/static-analysis/semgrep/00-installation.md [hunk 2]
```diff
@@ -211,9 +211,18 @@ according to your preferences and tooling.
...
+
+       ```sh
+       semgrep -c p/default --sarif --output scan_results.sarif
+       ```
+
      - **VIM format**: Use the VIM format to have all the information about a finding in a single line, making it
-      convenient for users of the Vim text editor.
+       convenient for users of the Vim text editor. Example usage of the `p/default` ruleset with the VIM output file:
```
Comment (fcasal) [inline content/docs/static-analysis/semgrep/00-installation.md:221]: same here, output file -> output format

**Review by fcasal — APPROVED** [review]
Output format seems better. Other than that, looks good to me

File: content/docs/static-analysis/semgrep/00-installation.md (hunk 2 above)
Comment (ahpaleus) [reply content/docs/static-analysis/semgrep/00-installation.md]: [Updated](https://github.com/trailofbits/testing-handbook/pull/28/commits/9fdffbd89ef35a273a1d62b85fdd3253ba95f475)

File: content/docs/static-analysis/semgrep/00-installation.md (hunk 1 above)
Comment (ahpaleus) [reply content/docs/static-analysis/semgrep/00-installation.md]: https://github.com/trailofbits/testing-handbook/pull/28/commits/9fdffbd89ef35a273a1d62b85fdd3253ba95f475

**Pull request merged**